RELEASE/scripts/autoscend/auto_familiar.ash

```diff
@@ -673,3 +674,65 @@ void preAdvUpdateFamiliar(location place)
 		}
 	}
 }
+
+boolean hatchFamiliar(item hatchling, familiar adult)
```

Comment:
you can just pass the familiar as the hatching can be derived from the $familiar record's hatchling field e.g.

```
> ash $familiar[Mosquito].hatchling

Returned: mosquito larva
```

Reply:
change made